ArduCopter/AP_Arming.cpp

```diff
@@ -298,6 +298,13 @@ bool AP_Arming_Copter::motor_checks(bool display_failure)
         return false;
     }
 
+	    //servo_test check
+#if FRAME_CONFIG == HELI_FRAME
+    if(copter.motors->servo_test_running()) {
+        check_failed(display_failure, "Servo Test is still running!");
```

Comment:
Let's remove the exclamation point at the end of the comment if that's ok.

Reply:
Ha, I had just done that before I even saw this, thanks @rmackay9 I agree.  That was a carry-over from when it said "Don't pull a Josh!"